app/pb_kits/playbook/pb_radio/_radio.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_tag(:label,
+    id: object.id,
+    data: object.data,
+    text: object.text,
+    value: object.value,
+    checked: object.checked,
+    class: object.classname) do %>
+
+
+    <input type="radio" value="<%= object.value %>" name="<%= object.name %>" <%= object.selected %>>
```

Comment:
You should use a rails input tag. Example: 

`<%= radio_button_tag object.name, object.value, object.checked, id: object.html_for, disabled: object.disabled %>`